tests/cpp/test_host_ir_jit.cpp

```diff
@@ -34,29 +33,47 @@ TEST_F(HostIrJitTest, Allocate) {
   hic->addInput(hic_in);
   hic->addOutput(hic_out);
 
-  // Adjust the number of allocates and calls to each allocate
-  int num_allocates = 10;
-  int num_calls_per_allocate = 10;
-  for (int i = 0; i < num_allocates; i++) {
-    auto* allocate =
-        IrBuilder::create<kir::Allocate>(hic_out, MemoryType::Global);
-    hic->pushBackTopLevelExprs(allocate);
-  }
+  hic->pushBackTopLevelExprs(hic_out->definition());
+
+  HostIrJit jit(std::move(hic));
+  auto options = at::TensorOptions().dtype(at::kFloat).device(at::kCUDA, 0);
+  at::Tensor in = at::randn({32, 16}, options);
+
+  KernelArgumentHolder in_args;
+  in_args.setCacheId(0);
+  in_args.push(in);
+  KernelArgumentHolder outs = jit.runWithInputs(in_args);
+  auto out = outs[0].as<at::Tensor>();
+  EXPECT_EQ(out.sizes(), in.sizes()) << "Sizes are not equal:\n"
+                                     << "in = " << in << "\n"
+                                     << "out = " << out;
+  EXPECT_EQ(out.strides(), in.strides()) << "Strides are not equal:\n"
+                                         << "in = " << in << "\n"
+                                         << "out = " << out;
```

Comment:
```suggestion
  EXPECT_EQ(out.strides(), in.strides());
```
EXPECT_EQ prints out LHS and RHS if not equal.